### DISCUSSION TITLE
Longer attack

### DISCUSSION THREAD

**Discussion by Jonatdel** [SynthstromAudible/DelugeFirmware#645] [Ideas]
I think it should be quite simple to implement a (much) longer attack range than the actual one with is ridiculously short. what do you think ?

**Comment by worker-gray** [comment]
i would add-- clock synched with time divisions would be a great enhancement as well.

**Comment by ok-reza** [comment]
did you know that you can patch attack to env1? if attack is at 50 and is modulated by env1 by a negative amount, your attack length can be a lot longer. set that modulation to -30 for example. this works for each ADSR separately too, so you can adjust the curves and length of decay, release, etc.
